app/views/content/terms_provider.md

```diff
@@ -0,0 +1,73 @@
+Thank you for taking part in trialling Manage teacher training applications. Here’s what you can expect from us, and what we ask of you, over the course of the pilot.
+
+## What you can expect from us
+
+1. ### Designated support
+
+    We will have designated support to answer your queries. You can contact us by:
+    * phone: INSERT TELEPHONE NUMBER
```

Comment:
There’s a card in Trello to review these policies before we launch – `INSERT TELEPHONE NUMBER` is one such thing we need to ensure we have in place.